### PR TITLE
added joinus

### PR DESCRIPTION
**Pull request by Ganesh-P03**

### CODE REVIEW

File: src/pages/about.js [hunk 1]
```diff
@@ -104,12 +146,41 @@ const About = () => {
         ))}
         </div>
       </> */}
-      </Container>
+          <form
+            action="https://docs.google.com/forms/d/e/1FAIpQLSeVXEGB_Kp_q7ZmUmkL3W5JPUNRv77jrvL6GMqlCcpP-N1inw/viewform"
+            target="_blank"
+          >
+            <Button>join us</Button>
```
Comment (NobleMathews) [inline src/pages/about.js:153]: Change to "JOIN US !", also the font doesn't seem to fit the rest of the page i feel could you have a look at that ?

File: src/pages/about.js [hunk 2]
```diff
@@ -104,12 +146,41 @@ const About = () => {
...
   );
 };
 
+const Button = styled.button`
+  all: initial;
+  cursor: pointer;
+  display: block;
+  float: right;
+  background-color: var(--color-blue);
+  padding: var(--sizing-sm) var(--sizing-base);
```
Comment (NobleMathews) [inline src/pages/about.js:167]: You might have t to change the padding used, currently if the row is full there is not enough spacing (ref https://i.imgur.com/EtmmZTW.png)

**Review by NobleMathews — CHANGES_REQUESTED** [review]
Nice work! please fix the 2 issues I highlighted it should good to go then

File: src/pages/about.js (hunk 1 above)
Comment (Ganesh-P03) [reply src/pages/about.js:153]: changed the text and font family!

File: src/pages/about.js (hunk 2 above)
Comment (Ganesh-P03) [reply src/pages/about.js:167]: Adjusted it

**Pull request merged**